app/src/docs/v1.api-spec.yaml

```diff
@@ -309,32 +480,55 @@ components:
             - low
             - high
           example: normal
+        subject:
+          type: string
+          description: The email subject
+          example: 'Hello {{ someone }}'
+    MergeResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/TransactionResponse'
+    MessageAssociation:
+      type: object
+      required:
+        - msgId
+        - to
+      properties:
+        msgId:
+          type: string
+          description: A corresponding message uuid
+          format: uuid
+          example: 00000000-0000-0000-0000-000000000000
         to:
           type: array
           items:
             type: string
-          description: >-
-            An array of recipients email addresses that will appear on the To:
-            field
+          description: An array of recipient email addresses that this message will go to
           example:
             - baz@gov.bc.ca
-        subject:
-          type: string
-          description: The email subject
-          example: Hello user
-    EmailMergeRequest:
+    MessageObject:
       type: object
       required:
         - bodyType
         - body
-        - contexts
         - from
+        - to
         - subject
       properties:
         attachments:
           type: array
           items:
-            $ref: '#/components/schemas/Attachment'
+            $ref: '#/components/schemas/AttachmentObject'
+          description: A list of recipient emails
```

Comment:
that's not right...  list of attachments.  but let's get the spec out, then clean up descriptions once we have the structures and names correct.

Reply:
Fixed the copy-paste mistake on the description